packages/forms-web-app/src/controllers/householder-planning/eligibility/enforcement-notice-householder.js

```diff
@@ -0,0 +1,81 @@
+const logger = require('../../../lib/logger');
+const { createOrUpdateAppeal } = require('../../../lib/appeals-api-wrapper');
+const {
+  VIEW: {
+    HOUSEHOLDER_PLANNING: {
+      ELIGIBILITY: { ENFORCEMENT_NOTICE_HOUSEHOLDER: currentPage },
+    },
+  },
+} = require('../../../lib/householder-planning/views');
+const {
+  validEnforcementNoticeOptions,
+} = require('../../../validators/householder-planning/eligibility/enforcement-notice-householder');
+const getPreviousPagePath = require('../../../lib/get-previous-page-path');
+
+const navigationPages = {
+  nextPage: '/before-you-start/claiming-costs-householder',
+  shutterPage: '/before-you-start/use-a-different-service',
+};
+
+exports.getEnforcementNotice = (req, res) => {
+  navigationPages.previousPage = getPreviousPagePath(req);
```

Comment:
This should be a specific page that's dictated in the ticket.

Reply:
This cannot be pointing to a static page as the user will be taken to this pager either from Decision Date or Date Decision Due page